src/plugins/plugin.title.js

```diff
@@ -121,8 +115,8 @@ class Title extends Element {
 		}
 
 		lineCount = helpers.isArray(opts.text) ? opts.text.length : 1;
-		textSize = lineCount * helpers.options._parseFont(opts).lineHeight + opts.padding * 2;
-
+		me._padding = helpers.options.toPadding(opts.padding);
+		textSize = lineCount * helpers.options._parseFont(opts).lineHeight + me._padding.height;
 		me.width = minSize.width = isHorizontal ? me.maxWidth : textSize;
 		me.height = minSize.height = isHorizontal ? textSize : me.maxHeight;
```

Comment:
I think the height needs to be updated to including top + bottom padding otherwise the box may be too small

Reply:
It looks like `textSize` includes the padding, so it's probably already taken into account? though maybe it should be added here instead of to `textSize` since it's not technically part of the text size